src/App.js

```diff
@@ -1,29 +1,10 @@
-import logo from './logo.svg';
+import ReactDOM from 'react-dom';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] What the previous reviewer said is not about the deleted code, it's about the component itself. In this milestone, The `App.js` component should be class-based. The objective is to practice the class-based React component.

Reply:
* The requested changes by the previous code reviewer is required for this project to be approved, it is stated in the project requirement to use class-based components for this project kindly fix this 🙏 
![image](https://user-images.githubusercontent.com/52497246/133337227-613781a5-5270-47eb-9c1b-5f13e7b47321.png)